README.md

```diff
@@ -37,13 +41,15 @@ See `man 5 xdg-desktop-portal-wlr`.
 
 ### Manual startup
 
-At the moment, some command line flags are available for testing, compatibility,
-or output selection. If you need to use one of these flags, you can provide an
-instance of xdpw using the following command:
+At the moment, some command line flags are available for development and
+testing. If you need to use one of these flags, you can start an instance of
+xdpw using the following command:
 
-```/usr/lib/xdg-desktop-portal -r & xdg-desktop-portal-wlr [OPTION...]```
+```sh
+/usr/lib/xdg-desktop-portal -r & xdg-desktop-portal-wlr -r [OPTION...]
```

Comment:
We probably don't even need to bother with the first half anymore (replacing the xdp instance). That was likely only needed to ensure it picked up a new xdpw instance. Now that xdpw is replaceable, we can probably just recommend `xdg-desktop-portal-wlr -r [OPTION...]`

LMK if you guys agree.

Reply:
Works for me.

Is there a minimum version of xdp for this to work?

Reply:
Ah right, good point.

> Is there a minimum version of xdp for this to work?

I don't think so

Reply:
I think this is actually a d-bus feature, so xdp version shouldn't matter.

https://www.freedesktop.org/software/systemd/man/sd_bus_request_name.html

Reply:
Eh, well, maybe xdp is doing something, but this was implemented 3 years ago...https://github.com/flatpak/xdg-desktop-portal/commit/0e67f33c7e296784019bce17b5684095791cbdfa